### PR TITLE
Add venue info and corresponding sections

### PR DESCRIPTION
**Pull request by willamesoares**
full-page

### CODE REVIEW

File: source/stylesheets/components/_navbar.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
   display: flex;
   position: fixed;
   width: 100%;
+  z-index: 1;
```
Comment (AlexRobertoCorrea) [inline source/stylesheets/components/_navbar.scss]: Is z-index really necessary here? Because it makes the CSS be more slow ...

Comment (alex-correa-dev) [inline source/stylesheets/components/_navbar.scss:6]: Is z-index really necessary here? Because it makes the CSS be more slow ...

**Comment by alex-correa-dev** [comment]
In my opinion, numbers like *Dias*, *Horas*, *Minutos* and *Segundos* could be centered, what do you think about?

**Comment by willamesoares** [comment]
@AlexRobertoCorrea By making the countdown label and numbers left and right aligned, respectively, I was trying to decrease the number of centralized items for this section since everything else is center aligned. 
I'll add a preview so you can see if that looks better. Your suggestion is to center align only the numbers, right?

**Comment by willamesoares** [comment]
@AlexRobertoCorrea 
centered_numbers_hero

**Comment by alex-correa-dev** [comment]
For me, it's better!

**Review by alex-correa-dev — APPROVED** [review]

**Pull request merged**